packages/invoices/src/OutboundShipment/api/hooks.ts

```diff
@@ -53,6 +53,7 @@ export const useOutboundFields = <KeyOfInvoice extends keyof Invoice>(
 export const useIsOutboundDisabled = (): boolean => {
   const { status } = useOutboundFields('status');
   return (
+    status === InvoiceNodeStatus.Shipped ||
```

Comment:
should be disabled for shipped also?

Reply:
yeah, I would think so.
Also - the inbound version is `useIsInboundEditable` and one of them should change to be consistent

Reply:
Yeah you're right